src/scripts/main.js

```diff
@@ -1,3 +1,71 @@
 'use strict';
 
-// write code here
+const tbody = document.querySelector('table tbody');
+
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+
+const max = 10;
+const min = 2;
+
+const setButton = (el, state) => (el.disabled = state);
+
+appendRow.addEventListener('click', (e) => {
+  const rows = [...tbody.querySelectorAll('tr')];
```

Comment:
This code is querying the DOM for all 'tr' elements every time the event is triggered. This can be inefficient if there are many rows in the table. Consider storing the rows in a variable and updating it only when a row is added or removed.